README.md

```diff
@@ -31,6 +31,7 @@ connected OHDSI/CMD/Atlas databases via SQL queries.
       - [How to make curl with Auth](#how-to-make-curl-with-auth)
       - [How to see the logs](#how-to-see-the-logs)
       - [In case of infra / network issues:](#in-case-of-infra--network-issues)
+      - [Updating dependencies](#updating-dependencies)
```

Comment:
this part seems to be missing? Not sure what should go there.

Reply:
oh yeah i think that was just some accident, removing now.

Reply:
wait actually realized that the line is auto generated. There's a section for updating dependencies that the link was accidentally removed i think in some other commit.